modules/xmpp/Lobby.js

```diff
@@ -307,7 +315,7 @@ export default class Lobby {
         if (memberRoomJid) {
             const jid = this.lobbyRoom.members[memberRoomJid].jid;
             const msgToSend
-                = $msg({ to: this.mainRoom.roomjid })
+                = $msg({ to: this.getMainRoomJidFromRoomJid(this.mainRoom.roomjid) })
```

Comment:
So, what are we doing here? Requesting access to a breakout rooom which has the lobby enabled through the main room?